### DISCUSSION TITLE
deployment error starting today

### DISCUSSION THREAD

**Discussion by kennyma603** [vercel/community#2492] [Help]
### Summary

i created a new react typescript project from sandbox without touching anycode, https://codesandbox.io/s/gifted-monad-gg09q4?file=/src/App.tsx
 then deploy to vercel.
 
 deployment failed: 
![image](https://github.com/vercel/community/assets/1115456/b41826f5-8988-44b4-abfa-6804d5389ce1)

anyone know how to fix? thanks

**Comment by ObserverMoment** [comment]
I've just started getting the exact same error...

**Comment by ObserverMoment** [comment]
This indicates the issue is resolved but I am still not able to deploy and getting exactly the same weird HookWebpackError...

https://www.vercel-status.com/incidents/rn3z83ftyjnb

**Comment by ObserverMoment** [comment]
Fixed with webpack update: https://github.com/webpack/webpack/issues/17213

**Reply by kennyma603** [reply]
Thanks! build deployment works again now.
